pkgs/by-name/ca/candy-icons/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gtk3
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "candy-icons";
+  version = "unstable-2023-10-03";
```

```suggestion
  version = "unstable-2023-12-31";
```
Changed to the latest commit date.